src/test_tri_vert_orientation.cpp

```diff
@@ -0,0 +1,57 @@
+#include <Omega_h_macros.h>
+
+#include <Omega_h_adj.hpp>
+#include <Omega_h_for.hpp>
+#include <Omega_h_mesh.hpp>
+#include <cstdio>
+#include <Omega_h_file.hpp>
+
+bool is_ccw_oriented(Omega_h::Mesh& mesh) {
+  OMEGA_H_CHECK(mesh.dim() == 2);
+  bool ccw;
+  // find mesh orientation: node number orientation of faces
+  // clockwise or counter clockwise: 1 for clockwise, -1 for counter clockwise
+  const auto& face2nodes = mesh.ask_down(Omega_h::FACE, Omega_h::VERT).ab2b;
+  const auto& coords = mesh.coords();
+  Omega_h::Write<Omega_h::LO> ccw_face(mesh.nfaces(), 0, "ccw_face");
+
+  auto find_ccw_face = OMEGA_H_LAMBDA(const Omega_h::LO face) {
+    const auto faceVerts = Omega_h::gather_verts<3>(face2nodes, face);
+    const auto faceCoords = Omega_h::gather_vectors<3, 2>(coords, faceVerts);
+    // calculate using (y2 - y1) * (x3 - x2) - (y3 - y2) * (x2 - x1)
+    const bool cw = (faceCoords[1][1] - faceCoords[0][1]) *
+                            (faceCoords[2][0] - faceCoords[1][0]) -
+                        (faceCoords[2][1] - faceCoords[1][1]) *
+                            (faceCoords[1][0] - faceCoords[0][0]) >
+                    0;
+    ccw_face[face] = (!cw) ? 1 : -1;
```

Comment:
Is there a reason why you use 1 and -1? If I ask the question `is_this_counter_clockwise` and you tell me `-1` it seems less obvious to me than `0` which corresponds to `false`.